Misc/NEWS.d/next/Core and Builtins/2022-08-12-18-13-49.gh-issue-91210.AWMSLj.rst

```diff
@@ -0,0 +1 @@
+Improve error message when a parameter without a default value follows one with a default value and show the same message, even when the non-default/default sequence is preceded by positional-only parameters.
```

Comment:
```suggestion
Improve error message when a parameter without a default value follows one with a default value and shows the same message, even when the non-default/default sequence is preceded by positional-only parameters.
```

Reply:
Thanks for the review @pablogsal. `show` here refers to the PR fixing a difference in an error, it doesn't describe current behavior. Something like `Improve error message, and show the same message`.